virtual_ship/expedition/ship_config.py

```diff
@@ -0,0 +1,156 @@
+"""ShipConfig and supporting classes."""
+
+from __future__ import annotations
+
+from datetime import timedelta
+from pathlib import Path
+
+import yaml
+from pydantic import BaseModel, ConfigDict, Field, field_serializer
```

Comment:
Is it confusing that Field here is not the same Field object as Parcels uses? that caught me below... Why not import `pydantic` and then refer to `pydantic.Field(...)` etc below?

Reply:
done